content/en/docs/concepts/scheduling/kube-scheduler.md

```diff
@@ -78,13 +76,25 @@ Finally, kube-scheduler assigns the Pod to the Node with the highest ranking.
 If there is more than one node with equal scores, kube-scheduler selects
 one of these at random.
 
+The are two supported ways to configure the filtering and scoring behavior
+of the scheduler:
+
+1. Scheduling Policies and
+1. Scheduling Profiles.
+
+### Scheduling policies
 
-### Default policies
+A scheduling Policy can be used to specify the predicates and priorities that
+the scheduler runs to filter and score nodes, respectively.
 
-kube-scheduler has a default set of scheduling policies.
+A policy can be specified through a file (via `--policy-config-file` flag) or a
+ConfigMap (via `--policy-configmap` flag) that follows the
+[Policy type](https://pkg.go.dev/k8s.io/kube-scheduler@v0.18.0/config/v1?tab=doc#Policy).
 
 ### Filtering
```

Comment:
I think we should move the section `Filtering` and `Scoring` in section "Scheduling with kube-scheduler", right before introducing scheduler policies and profiles, like this:

## Scheduling with kube-scheduler

...

### FIltering

### Scoring

## Configuring kube-scheduler (or similar name)

### Scheduling policies

### Scheduling profiles

Reply:
I don't think so. Because those sections use the names (for Predicates and Priorities) that only the Policy API exposes.

Reply:
Or, we put a table for mapping Predicate/Priority name to Plugin name. Then guide the audience which literals to use upon different configuring manners (policy vs. profile). WDYT?

Reply:
The only caveat is that there is no direct mapping, because a predicate plus a priority might map to a single plugin.

What about using the subtitles `Predicates` and `Priorities` inside the `Scheduling Policies` section?

Reply:
Version https://5e66a3ce506312000a06f41d--kubernetes-io-vnext-staging.netlify.com/docs/concepts/scheduling looks good to me.